templates/Default/engine/Default/alliance_stat.php

```diff
@@ -16,12 +16,32 @@
 
 if ($CanChangeChatChannel) { ?>
 	<tr>
-		<td class="top">Discord Channel ID:&nbsp;</td>
-		<td><input type="text" name="discord" size="30" value="<?php echo htmlspecialchars($Alliance->getDiscordChannel()); ?>" /></td>
+		<td class="top">Discord Server ID:&nbsp;</td>
+		<td>
+			<input type="text" name="discord_server" size="30" value="<?php echo htmlspecialchars($Alliance->getDiscordServer()); ?>" />
```

Comment:
Is a size of 30 enough for discord servers? (I assume it probably is as I think they use short urls, but just in case)

Reply:
Yeah, the ID's are currently 18 characters longs, so 30 should be safe.

Reply:
ok